_harp/about/_company.md

```diff
@@ -9,3 +9,17 @@ We support ourselves entirely through voluntary contributions from our alumni, t
 We believe this to be a replicable model of training that is capable of addressing acute skills shortages, widening access to work opportunities, and offering life-changing personal empowerment free at the point of delivery and without ongoing subsidies.
 
 We are partnering with a number of organisations in the social enterprise and charity sectors to provide freelancing and employment opportunities for our graduates and to open new schools in the UK and abroad.
+
+## Our hiring rate
+
+Our employment success rate for students who have actively sought development work after graduation is 100%.
+
+Of the 52 students who studied with us in 2014/15 and who actively sought development work, all 52 are now working either freelance or in permanent roles. This is a reflection of both the quality of our graduates and the level of support we provide to them after they graduate.
```

Comment:
Removed one of the paragraphs and changed "development work" to "work as a web developer"